solr/core/src/test/org/apache/solr/servlet/CacheHeaderTest.java

```diff
@@ -59,7 +59,7 @@ public void testCacheVetoHandler() throws Exception {
             f.getCanonicalPath(),
             CommonParams.STREAM_CONTENTTYPE,
             "text/csv");
-    HttpResponse response = getClient().execute(m);
+    HttpResponse response = getHttpClient().execute(m);
```

Comment:
normally I wouldn't like the direction of this -- generic to HTTP specific, but here, clearly the test is related to HTTP.